File: Oficinas/29-07-22 Introdução à Programação Competitiva/Códigos/Non-Integer Donuts/donuts.py
import os

n = int(input())
str = input()
centavos = int(str[str.find(".") + 1 : len(str)])
atrasado = 0

for i in range(n):
    str = input()
    decimal = int(str[str.find(".") + 1 : len(str)])
    centavos = (centavos + decimal) % 100
    
    if centavos != 0:
        atrasado = atrasado + 1
        
print(atrasado)

File: Oficinas/29-07-22 Introdução à Programação Competitiva/Códigos/Non-Integer Donuts/donuts.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    char aux;
    int n, inteiro, decimal, centavos, atrasado = 0;
    cin >> n;
    cin >> aux >> inteiro >> aux >> decimal;
    centavos = decimal;

    for (int i = 0; i < n; i++) {
        cin >> aux >> inteiro >> aux >> decimal;
        centavos = (centavos + decimal) % 100;

        if (centavos != 0) {
            atrasado++;
        }
    }

    cout << atrasado << "\n";

    return 0;
}